storage/ndb/rest-server2/server/src/config_structs.hpp

```diff
@@ -32,8 +32,6 @@
 #define CM(DATATYPE, VARIABLENAME, JSONKEYNAME, INITEXPR, DOCSTRING) \
   DATATYPE VARIABLENAME;
 #define PROBLEM(CONDITION, MESSAGE)
-#define CLASSDEFS(...) __VA_ARGS__
-#define VECTOR(DATATYPE)
```

Comment:
These are used in `config_structs_def.hpp`, which is included below. Why should they be removed?

Reply:
Just tested compiling, and it fails due to this. After reverting this chunk, compilation succeeds.